src/com/magento/idea/magento2uct/ui/ConfigurationDialog.java

```diff
@@ -76,6 +74,7 @@ public ConfigurationDialog(final @NotNull Project project) {
         setTitle(ConfigureUctAction.ACTION_NAME);
         getRootPane().setDefaultButton(buttonOk);
 
+        hasAdditionalPath.addActionListener(event -> refreshAdditionalFields(hasAdditionalPath.isSelected()));
```

Comment:
Please, call this method `refreshAdditionalFields` in the constructor. So we will get saved state of this component.

Reply:
Pay attention, this method should be called after `setDefaultValues`.

Reply:
Thanks for your code review. I fixed. Please check.